### PR TITLE
Yet another random item fix PR

### PR DESCRIPTION
**Pull request by Karol1223**
#### Summary
None

#### Purpose of change
Consistency. Realism. My sanity.

#### Describe the solution
<details>
<summary>WEIGHT CHANGES</summary>

- Mess kit: 1.5kg -> 0.5kg
</details>

<details>
<summary>VOLUME CHANGES</summary>

- Mess kit: 1l -> 1.1l
</details>

<details>
<summary>MATERIAL CHANGES</summary>

- Mess kit: no longer has ``plastic`` as its material, instead it is now 100% ``aluminum``
</details>

<details>
<summary>DISASSEMBLY CHANGES</summary>

- Sheet: uses ``sheet_cotton`` in place of ``sheet_cotton_patchwork``; no longer has ``BLIND_EASY``
- Patchwork cotton sheet: given ``BLIND_EASY``
- ALL coins: disassembly recipes removed, they are too small to properly salvage and they are not made of copper (most are, if I recall, cupronickel)
- Silver Eagle coin: ``silver_small`` 31 -> 10
- American Buffalo coin: ``gold_small`` 31 -> 6
- MOLLE webbing belt: manual disassembly removed, now handled via reversible recipe
- Police duty belt: manual disassembly removed, now handled via reversible recipe
- Backpack: completely reworked, no longer gives patches, instead gives sheets and zippers, also uses different tools and takes 15min longer
- Leather backpack: time taken 14h -> 25min
- Purse: initial disassembly implementation
- Daypack: initial disassembly implementation
- Chest ammo pouch: time taken 2.5h -> 10min
- Duffel bag: time taken 14h -> 1h
- Holster: time taken 1h -> 10min
- Leather pouch: time taken 1h -> 10min
- Sheath: time taken 4h -> 25min
- Travelpack: time taken 7.5h -> 35min
- Petpack: time taken 6.66h -> 35min
- Large tactical backpack: time taken 18h -> 1h
- Hiking backpack: time taken 14h -> 1h
- Double rifle case: time taken 4.5h -> 25min
- Military rucksack: initial disassembly implementation
- MOLLE assault pack: initial disassembly implementation
- Messenger back: initial disassembly implementation
- Canvas sack: initial disassembly implementation
- Mess kit: completely reworked, now only needs ``HAMMER`` 1 and yields take 10 scrap aluminum, also takes less time
- All three sizes of scrap leg guards: handled via a reversible recipe to avoid generating mass (the recipe itself needs more tuning but that's beyond my paygrade)
</details>

<details>
<summary>RECIPE CHANGES</summary>

- Chunk of copper: ``copper`` 50 -> 32
- Copper: all recipes obsoleted, this can all be handled via disassembly
- Anesthesia kit: recipe obsoleted
- Extra light frame + Foldable extra light frame: now use ``welding_alloys`` instead of ``welding_standard``
- MOLLE webbing belt + Police duty belt: can no longer use strings or shoelaces in place of clasps
- Mess kit: recipe obsoleted
</details>

<details>
<summary>PRICE CHANGES</summary>

- 40x46mm M1006 sponge: ``pre-apo`` 100$ -> 23$; ``post-apo`` 60$ -> 0.05$ (this is a sponge projectile, effectively useless)
- Wire: ``post-apo`` 1$ -> 0.1$
- Set of pipe fittings: ``post-apo`` 5$ -> 0.25$
- Quilt: ``post-apo`` UNDEFINED -> 2$
- Anesthesia kit: ``post-apo`` 100$ -> 15$ (this may still be overkill, this item appears to be a source of much confusion among the devcord community and I have a hard time properly defining what it is supposed to represent)
- Welding blanket: ``post-apo`` UNDEFINED -> 5$
- Makeshift welding blanket: ``post-apo`` UNDEFINED -> 2$
- Mess kit: ``pre-apo`` 50$ -> 30$; ``post-apo`` 5$ -> 1$
- Multivitamin: ``pre-apo`` 5$ -> 0.9$; ``post-apo`` 20$ -> 1$ (these are prices for ONE PILL)
</details>

<details>
<summary>OTHER CHANGES</summary>

- Mess kit: no longer an electronic tool - there don't appear to be electric mess kits irl so instead I made it one of the common Amazon ones. It's basically a fancy pot now.
- Military mess kit: obsoleted
- L-stick: obsoleted
- As many deactivated and destroyed robots as I could find have been given ``NO_SALVAGE`` to avoid weird cutting up with kitchen knives
- Mess tin: migrated into a mess kit
</details>

#### Additional context
These have become my favorite PRs to do very quickly. Throw more items that need fixes at me.

### CODE REVIEW

**Comment by Maleclypse** [comment]
I noticed that you removed c++ relating to the military mess kit but put the kit in Aftershock. Won’t that mean that the kits in Aftershock are now broken?

**Comment by Karol1223** [comment]
> I noticed that you removed c++ relating to the military mess kit but put the kit in Aftershock. Won’t that mean that the kits in Aftershock are now broken?

Two things, but I am not sure about either
1. I believe the test specifically referenced the regular mess kit, I don't see it connecting to the military one though I may be blind and/or stupid. I've only started C++ two days ago so I may be wrong.
2. I believe, though again could likely be very wrong, the test was... just a test. I don't think these tests have any presence or effect in the actual game. But again, could be wrong.

**Comment by Maleclypse** [comment]
> > I noticed that you removed c++ relating to the military mess kit but put the kit in Aftershock. Won’t that mean that the kits in Aftershock are now broken?
> 
> Two things, but I am not sure about either
> 
> 1. I believe the test specifically referenced the regular mess kit, I don't see it connecting to the military one though I may be blind and/or stupid. I've only started C++ two days ago so I may be wrong.
> 2. I believe, though again could likely be very wrong, the test was... just a test. I don't think these tests have any presence or effect in the actual game. But again, could be wrong.

That’s my bad. Reviewing gets more difficult with that many files touched. You are correct it is test data only.

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -1315,8 +1315,9 @@
     "description": "Essential dietary nutrients conveniently packaged in pill form.  An option of last resort when a balanced diet is not possible.  Excess use can cause hypervitaminosis.",
     "weight": "1 g",
     "volume": "100 ml",
-    "price": 500,
-    "price_postapoc": 2000,
+    "price": 90,
+    "price_postapoc": 200,
```
Comment (Maleclypse) [inline data/json/items/comestibles/med.json:1319]:
```suggestion
    "price_postapoc": 20,
```

**Review by Maleclypse — APPROVED** [review]
Thanks for doing this!

**Comment by Karol1223** [comment]
Are you sure we want multivitamins to be cheaper post-apo than pre-apo? They're an irrecoverable consumable and that's why I made them 2$ per pill. I think 20¢ is too cheap there, honestly?

**Comment by Karol1223** [comment]
@Maleclypse pinging to ask because I forgot to do so in the previous message, sorry

**Comment by Maleclypse** [comment]
> @Maleclypse pinging to ask because I forgot to do so in the previous message, sorry

I was attempting to get to a bottle cost of around $2 but my main goal was to make sure that a multivitamin wasn't more expensive than a unit of irreplaceable recreational drugs like cocaine. Karol and I have spoken about this on discord and I support whatever choice they make.

**Comment by Maleclypse** [comment]
Ping me on discord if you are unable to rebase or resolve the conflicts.

**Pull request merged**